src/js/components/Carousel/carousel.stories.js

```diff
@@ -22,4 +38,27 @@ const SimpleCarousel = () => (
   </Grommet>
 );
 
-storiesOf('Carousel', module).add('Simple Carousel', () => <SimpleCarousel />);
+const CustomCarousel = () => (
```

Comment:
Overall the direction is great, THANKS!
I will suggest removing the usage of ThemeContext, it's a better practice to use it when wanting to override and apply a custom theme, in this case, we don't really override and can apply the theme on the Grommet component, see my suggested commits

Reply:
@ShimiSun ok, my idea was to show how we can extend the theme, but for sure, it makes more sense to pass the theme into the Grommet Component here